DemoApp/SceneDelegate.swift

```diff
@@ -34,6 +34,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         self.window = window
         window.makeKeyAndVisible()
         scene.windows.forEach { $0.tintColor = .streamBlue }
+        
+        if let userId = UserDefaults(suiteName: applicationGroupIdentifier)?.string(forKey: currentUserIdRegisteredForPush),
+           let userCredentials = UserCredentials.builtInUsersByID(id: userId) {
+            coordinator.presentChat(userType: .credentials(userCredentials))
+        }
```

Comment:
I would add `start` method to `DemoAppCoordinator` and make this check inside

Reply:
good idea

Reply:
That's a good time to refactor that part 🙂 If previously we didn't have any logic on startup, now we have. Launching from storyboard is no longer the way

Reply:
> If previously we didn't have any logic on startup, now we have. 

It's literally 3 new lines added, we can't call this "we didn't have any logic, now we have" IMHO. `DemoAppCoordinator` needs a refactor for a proper `start` func, which would go over the scope of this PR.

Reply:
I feel you but we never get back to fix tech debt in `DemoApp` 😄 it's like a snowball. 

The demo is the first thing devs check and it contributes to the first impression so spending some time making it look nice is worth it but that's just me.

Reply:
@GetStream/ios-developers WDYT?